src/components/NavBar.tsx

```diff
@@ -28,6 +28,20 @@ export default function NavBar() {
             />
           </a>
         </Link>
+        <div className="flex justify-between items-center">
+          <Link passHref href="/">
+            <span className="mx-3 cursor-pointer">Home</span>
```

Comment:
These `span`'s should be anchor tags. Then you can remove the `passHref`'s prop and the `cursor-pointer` class.